framework/build.gradle

```diff
@@ -20,6 +20,12 @@ ext {
     apply from: 'cordova.gradle'
     cdvCompileSdkVersion = privateHelpers.getProjectTarget()
     cdvBuildToolsVersion = privateHelpers.findLatestInstalledBuildTools()
+    if (project.hasProperty('cdvMinSdkVersion')) {
```

Comment:
Shouldn't we guard it with a check that the given string is indeed an integer?

Reply:
I added an isInteger check: https://github.com/apache/cordova-android/pull/1117/files#diff-3dc11ca3a71905f437e1e170f963e9854da27943efcff23bf87bfe87abeda513R23

But, I didn't see any other guards in the cordova gradle files. I might have missed it off course as I am no Gradle expert. 
IMO, if we want to assist the users with additional type checking, it should be in a seperate routine, and the wrong types must be logged with warning or error to the user (if that is possible).